app/src/common/urls.js

```diff
@@ -258,6 +258,7 @@ export const URLS = {
   githubAuthSettings: () => `${uatBase}settings/oauth/github`,
   analyticsServerSettings: () => `${urlBase}settings/analytics`,
   events: (projectKey) => `${urlBase}${projectKey}/activity`,
+  eventsController: () => `${urlBase}activities/searches`,
```

Comment:
I suppose the bug occurred as we didn't apply the changes from [this PR](https://github.com/reportportal/service-ui/pull/3536) correctly while merging develop into `feature/orgs`.
So please reflect the changes from the mentioned PR (no need to create the new controller, just replace the URL with the new one and use it correctly in `events/sagas`).